UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs

```diff
@@ -73,6 +85,18 @@ public void AddDrunkTime(float time)
 			}
 		}
 
+		public void BlindnessValue(bool IsBlind)
+		{
+			if (IsBlind)
+			{
+				Camera.main.GetComponent<CameraEffects.CameraEffectControlScript>().LightingSystem.fovDistance = 0.65f;
```

Comment:
make the numbers either a constant or serialized field.

Reply:
looks like this accidentally got included